docs/reference_guides/model_libraries/models_extra/membrane_model/1d_membrane.rst

```diff
@@ -10,5 +10,30 @@ and sweep side. The sweep stream inlet is optional. The driving force across the
 partial pressure difference in this gas separation application. Additionally, the energy balance
 assumes that temperature remains constant on each side of the membrane.
 
+Variables
+---------
+
+Model Inputs - symbol:
+
+* Membrane length - :math:`L`
+* Membrane Area - :math:`A`
+* Permeance - :math:`per`
+* Feed flowrate - :math:`F_fr`
+* Feed compositions - :math:`x`
+* Feed pressure - :math:`P`
+* Feed temperature - :math:`T`
```

Comment:
Should selectivity be here, or is that calculated from permeances?

Also, should there be mention of sweep conditions here too?

Reply:
Thanks! The selectivity could be calculated and sweep condition is mentioned in the beginning.